sktime/transformations/hierarchical/reconcile.py

```diff
@@ -61,7 +66,7 @@ class Reconciler(BaseTransformer):
 
     METHOD_LIST = ["bu", "ols", "wls_str"]
 
-    def __init__(self, method="bu"):
+    def __init__(self, method="ols"):
```

Comment:
we cannot change the default without deprecation now. The reconcilers are being used since 0.11.2, so this change would require a deprecation message, which is displayed whenever `"bu"` is used. Are we certain we want this change in default?

Reply:
Good point, I think the `"bu"` is the more intuitive as well - changed back for now. Although `"ols"` may be the preferred method for users in the long-term since it *typically* is more skillful - forecasts at all levels are used in the reconciliation process.